src/NATS.Client.JetStream/INatsJSContext.cs

```diff
@@ -204,4 +204,12 @@ public interface INatsJSContext
     IAsyncEnumerable<INatsJSStream> ListStreamsAsync(
         string? subject = default,
         CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// List stream names.
+    /// </summary>
+    /// <param name="subject">Limit the list to streams matching this subject filter.</param>
+    /// <param name="cancellationToken">A <see cref="CancellationToken"/> used to cancel the API call.</param>
+    /// <returns>Async enumerable list of stream names to be used in a <c>await foreach</c> loop.</returns>
+    IAsyncEnumerable<string> ListStreamNamesAsync(string? subject = default, [EnumeratorCancellation] CancellationToken cancellationToken = default);
```

Comment:
This was missed out. thanks.
Not sure if we need the annotation on the interface: `The 'EnumeratorCancellation' attribute applied to parameter 'cancellationToken' has no effect. The attribute is only effective on a parameter of type 'CancellationToken' in an async-iterator method returning 'IAsyncEnumerable<>'.`